cobol-parser/src/main/scala/za/co/absa/cobrix/cobol/parser/examples/generators/TestDataGen13bCompaniesFileHeaders.scala

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.cobrix.cobol.parser.examples.generators
+
+import java.io.{BufferedOutputStream, FileOutputStream}
+
+import scodec.Attempt.Successful
+import za.co.absa.cobrix.cobol.parser.decoders.BinaryUtils
+import za.co.absa.cobrix.cobol.parser.examples.generators.model.Company
+
+import scala.util.Random
+
+/**
+  * This is a test data generator. The copybook for it is listed below.
+  * This generates a data file having an big endian RDW header.
+  *
+  * The output file will contain a file headerof 100 bytes and footer of 120 bytes.
+  */
+object TestDataGen13bCompaniesFileHeaders {
+
+  val numberOfRecordsToGenerate = 20
+
+  // seed=100 is used for the integration test
+  val rand: Random = new Random(/*100*/)
+
+  /*
+          01  COMPANY-DETAILS.
+              05  SEGMENT-ID        PIC X(5).
+              05  COMPANY-ID        PIC X(10).
+              05  STATIC-DETAILS.
+                 10  COMPANY-NAME      PIC X(15).
+                 10  ADDRESS           PIC X(25).
+                 10  TAXPAYER.
+                    15  TAXPAYER-TYPE  PIC X(1).
+                    15  TAXPAYER-STR   PIC X(8).
+                    15  TAXPAYER-NUM  REDEFINES TAXPAYER-STR
+                                       PIC 9(8) COMP.
+              05  CONTACTS REDEFINES STATIC-DETAILS.
+                 10  PHONE-NUMBER      PIC X(17).
+                 10  CONTACT-PERSON    PIC X(28).
+   */
+
+  def putStringToArray(bytes: Array[Byte], str: String, index0: Int, index1: Int): Unit = {
```

Comment:
Seems to be a duplicate of the `TestDataGen13aFileHeaderAndFooter `.

Reply:
Yeah, it might be a good idea to extract all data generation helpers in a utility package. Will do it as a separate task. Thanks.